SS_readstarter.tpl

```diff
@@ -49,6 +49,7 @@
   int k1
   int k2
   int k3
+  int special_flag
```

Comment:
special_flag doesn't strike me as an informative name...maybe change to something more informative?